docs/rfc/001-backup-format.md

```diff
@@ -0,0 +1,39 @@
+# Structured Backup format
+
+## Principles
+
+- Based-zip64 format (compression level: archive)
+- Based-zip64 standard file encryption
+- Structured tree structure
```

Comment:
Should we have an integrity mechanism?

Reply:
generate all `manifest-*.signed` file hash to one result (sha2-512, sha3-512) to remote logging

```
if query the logging exists then
   verified
else
   unverified
```